dokka-integration-tests/cli/build.gradle.kts

```
@@ -15,14 +15,18 @@ dependencies {
     implementation(projects.utilities)
 }
 
-/* Create a fat base plugin jar for cli tests */
-val basePluginShadow: Configuration by configurations.creating {
+// Configuration for plugins/dependencies required to run CLI with base plugin
+val cliPluginsClasspath: Configuration by configurations.creating {
```

```suggestion
val cliPluginsClasspath: Configuration by configurations.creating {
    description = "plugins/dependencies required to run CLI with base plugin"
```

Suggestion: use description to document the usage, so it's visible in more places.